src/core/stream/representation/representation_stream.ts

```diff
@@ -209,8 +210,22 @@ export default function RepresentationStream<TSegmentDataType>(
     segmentsLoadingCanceller.signal,
   );
 
+  const canStream = new SharedReference<boolean>(true);
+
+  playbackObserver.listen((observation) => {
+    const observationCanStream = observation.canStream ?? true;
+    if (canStream.getValue() !== observationCanStream) {
+      log.debug("Stream: observation.canStream updated to", observationCanStream);
+      canStream.setValue(observationCanStream);
+    }
+  });
```

Comment:
We now need to add a `clearSignal` parameter for this I think (I guess here for a good reason?)

Reply:
Yes, done